src/main/java/core/basesyntax/Draw.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Draw {
+    public abstract String getDraw();
```

Comment:
```suggestion
    String getDraw();
```
`public abstract` are default values for methods in the interface so that we can remove them